packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx

```diff
@@ -125,7 +125,7 @@ export class GroupedListBase extends BaseComponent<IGroupedListProps, IGroupedLi
     }
   }
 
-  private _renderGroup = (group: any, groupIndex: number): JSX.Element | null => {
+  private _renderGroup = (group: IGroup | undefined, groupIndex: number): JSX.Element | null => {
```

Comment:
Minor point, but since this is a private function, another option is `(groupIndex: number, group?: IGroup)`

Reply:
That's a great point and an API improvement :)

Reply:
Actually, this specific call is bound to `List#onRenderCell`. So for the time-being let's leave this specific function parameter order as-is and just try to improve the types as we prepare for refactor.

Reply:
I do agree with you though 😄.

Reply:
If they are bound, can you use a lookup type? Like `private _renderGroup: IList['onRenderCell'] = (group, groupIndex)`? TS3.x will help expose inconsistencies in these areas, but I also think there's value in explicitly using the `onRenderCell` type here.

Reply:
(I also notice now the return types aren't technically matching.. another thing I wonder if TS3.x would catch.)

Reply:
> (I also notice now the return types aren't technically matching.. another thing I wonder if TS3.x would catch.)

I addressed this in https://github.com/OfficeDev/office-ui-fabric-react/pull/8043/commits/e15b171853defa90d56d734d67a97ee21dee2c76 (good catch).

For the lookup type, let's shelve that for now given:

https://github.com/OfficeDev/office-ui-fabric-react/blob/f8deb819680c93729495956f900b11a9e7bf3753/packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.base.tsx#L85